packages/chaire-lib-backend/src/services/mailer/transport.ts

```diff
@@ -5,14 +5,48 @@
  * License text available at https://opensource.org/licenses/MIT
  */
 import nodemailer, { Transporter } from 'nodemailer';
+import { _isBlank } from 'chaire-lib-common/lib/utils/LodashExtensions';
+import fs from 'fs';
+import DKIM from 'nodemailer/lib/dkim';
+
+// Generate the config block for DKIM if the right ENV variables are defined
+const getDkimConfig = (): DKIM.Options | undefined => {
```

Comment:
You probably know exactly what dkim is, but for readers neophyte in email configuration, an additional sentence explaining why they should google dkim to have more information about it would be helpful. So just why we need a dkim config, what problem it fixes.